Code/GraphMol/SmilesParse/smarts.yy

```diff
@@ -93,22 +95,43 @@ namespace {
 %left AND_TOKEN
 %right NOT_TOKEN
 
+%destructor { delete $$; } ATOM_TOKEN
```

Comment:
cool, I didn't know about this.

Reply:
I found it in the docs, while trying to solve the problems with the parsers. I only managed it partially, and, since I am not too sure about what I tried, I did not update the *.tab.cpp.cmake files.